scripts/config/pages.pl

```diff
@@ -116,6 +116,8 @@ sub all_pages {
         ['user/security/self_exclusionws',   'user/security/self_exclusion',         'default',    'Self Exclusion'],
         ['user/settings/assessmentws',       'user/settings/financial_assessment',   'default',    'Financial Assessment',    'ja'],
         ['user/settings/detailsws',          'user/settings/personal_details',       'default',    'Personal Details'],
+        # TODO: uncomment this code if we allow clients to claim professional trader
```

Comment:
yes, we do need to allow clients to request professional status. e.g. we will have Australian CR clients who will want to upgrade to professional so they can subscribe to the ICO